images/linux/scripts/installers/ruby.sh

```diff
@@ -1,13 +1,22 @@
 #!/bin/bash -e
 ################################################################################
 ##  File:  ruby.sh
-##  Desc:  Installs Ruby requirements
+##  Desc:  Installs Ruby requirements and ruby gems
 ################################################################################
 
 source $HELPER_SCRIPTS/install.sh
 
-sudo apt-get install ruby-full
-sudo gem update --system
+apt-get install ruby-full
+gem update
+
+# Install ruby gems from toolset
+gemsToInstall=$(get_toolset_value ".rubygems[] .name")
+if [ -n "$gemsToInstall" ]; then
+    for gem in $gemsToInstall; do
+        echo "Installing rubygem $gem"
```

Comment:
RubyGem is a tool name, the package itself is called gem, so I think the message should be "Installing gem"

Reply:
makes sense